README.md

```diff
@@ -28,6 +28,72 @@ When upgrading to this version of Debugger for Mainframe from an older version,
 #### Client
 - Java version 8.0 or higher.
 
+### Set up Secure Connection to InterTest Server
+
+#### Prerequisites
+- InterTest server running and configured to support secure communication.
+- Security certificate for the InterTest server.
+
+**Follow these steps**
+
+1. Download the Server Certificate to your PC
+
+2. Import certificate with entry name HOSTNAME to the trust store, for example:
+
+    *jdk0.0.0_000\abc\Lib\security* named cacerts of the JRE under which the Debugger for Mainframe DA client is running.
+
+3. Proceed using Command Line, or a UI Tool:
+
+    - Command Line:
+    
+      Enter the following command:
+      
+         *sudo keytool -import -alias hostname -file hostname.cer -storetype JKS -keystore cacerts*
+
+    - UI:
+        1. In your preferred UI, locate and open cacerts (pw: **changeit**)
+        2. Import the certificate to cacerts
+        3. Name the certificate with an appropriate alias to ensure it is easily identified
+        
+    - Linux Subsystem
+        1. Verify java is installed: java -version
+        2. Locate your subsystem's java installation
+        3. Go to /lib/security to find **cacerts**
+        4. Import the certificate :
+        
+            sudo keytool -import -alias hostname -file hostname.cer -storetype JKS -keystore cacerts
+        
+4. Open VS Code and try to debug something while your launch.json looks like this:
+    
+        {
+                "type": "cbl",
+                
+                "request": "launch",
+                
+                "name": "Cobol Intertest CICS Debug",
+                
+                "programName": "PROGNAME",
+                
+                // "debugServer": "Server ID",
+                
+                "interTestHost": "HOST URL",
+                
+                "interTestPort": 12345,
+                
+                "interTestUserName": "PMFID",
+                
+                "interTestSecure": true,
+                
+                "cicsApplId": "CICSappID"
+            }
+        
+Note: **interTestSecure": true** will display if Secure connection has been succesfully configured.
```

Comment:
This step is a bit confusing. Shouldn't it be:
4. Check launch.json looks like this / has "interTestSecure: true"
5. Run a test debug session